mina.netty/src/main/java/org/jboss/netty/channel/socket/nio/AbstractNioWorker.java

```diff
@@ -618,4 +643,53 @@ public void run() {
             }
         });
     }
+
+    public void register(NioChildDatagramChannel childChannel) {    // TODO no datagram specific
```

Comment:
I want to combine register/deregister with the existing methods that take AbstractNioChannel. But it registers the channel with a Selector which we don't require for child channels. I need feedback here.